packages/eventual-send/src/index.js

```diff
@@ -1,15 +1,23 @@
+// @ts-check
```

Comment:
Now that we extracted separate local.js and postposed.js files, I'd prefer if we followed our general convention for multi-module packages: Move this to a something.js and then have an index.js that just does the `export from` stuff.

We seem to be split about whether these re-exporting index.js files are inside or outside of src/ . For this package there's a related mystery: There's a shim.js outside of src/ but a no-shim.js *inside* src/ . Why?

Oh, I see now that both shim.js and no-shim.js are functioning as index.js files normally do. So no need for another one. But IIUC, this index.js file should then be named something else. But what I don't understand: What's the relationship between an index.js file and an index.d.ts file?

Reply:
> I can rename index.js, but I'd rather not do that until the rest of the review is finished.

ok

Reply:
Please remember this renaming before merging.

Reply:
Renamed `index.js` to `handled-promise.js`.